graph/impl/KokkosGraph_Distance2MIS_impl.hpp

```diff
@@ -828,7 +828,6 @@ struct D2_MIS_FixedPriority {
     Kokkos::parallel_for(range_pol(0, numVerts),
                          InitWorklistFunctor(worklist1));
     lno_t workRemain = numVerts;
-    int numIter      = 0;
     while (workRemain) {
```

Comment:
Why are there changes in Distance2? I am assuming to remove warnings. If yes then ok.

Reply:
This is to remove warnings.